### PR TITLE
Add support for optional control data analog to nf-core/chipseq

### PR DESCRIPTION
**Pull request by bjlang**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/atacseq/tree/master/.github/CONTRIBUTING.md)- [ ] If necessary, also make a PR on the nf-core/atacseq _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [x] Usage Documentation in `docs/usage.md` is updated.
- [x] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [x] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: bin/check_samplesheet.py [hunk 1]
```diff
@@ -34,7 +35,7 @@ def print_error(error, context="Line", context_str=""):
     sys.exit(1)
 
 
-def check_samplesheet(file_in, file_out):
+def check_samplesheet(file_in, file_out, expect_control=False):
```
Comment (drpatelh) [inline bin/check_samplesheet.py:38]:
```suggestion
def check_samplesheet(file_in, file_out, with_control=False):
```

Comment (drpatelh) [reply bin/check_samplesheet.py:38]: To be consistent with `params.with_control`

File: conf/test.config [hunk 2]
```diff
@@ -20,7 +20,8 @@ params {
     max_time   = '6.h'
 
     // Input data
-    input       = 'https://raw.githubusercontent.com/nf-core/test-datasets/atacseq/samplesheet/v2.0/samplesheet_test.csv'
+    // input       = 'https://raw.githubusercontent.com/nf-core/test-datasets/atacseq/samplesheet/v2.0/samplesheet_test.csv'
+    input = "$projectDir/assets/test.csv" // TODO delete file and push it to datesets
```
Comment (drpatelh) [inline conf/test.config:24]: This needs to be fixed?

File: docs/usage.md [hunk 3]
```diff
@@ -38,31 +38,37 @@ CONTROL,AEG588A1_S1_L004_R1_001.fastq.gz,AEG588A1_S1_L004_R2_001.fastq.gz,1
...
 
+### Control data
+
+If controls are to be used for peak calling use the parameter `--with_control`. In this case, the samplesheet file needs the additional columns `control` and `control_replicate`. These should be the sample identifier and sample replicate for the controls.
+
 ### Full samplesheet
 
 The pipeline will auto-detect whether a sample is single- or paired-end using the information provided in the samplesheet. The samplesheet can have as many columns as you desire, however, there is a strict requirement for the first 4 columns to match those defined in the table below.
 
 A final samplesheet file consisting of both single- and paired-end data may look something like the one below. This is for 7 samples, where we have biological triplicates for both the `CONTROL` and `TREATMENT` groups, and the third replicate in the `TREATMENT` group has been a technical replicate as a result of being sequenced twice.
```
Comment (JoseEspinosa) [inline docs/usage.md:49]: This text was so before? 👀

File: nextflow_schema.json [hunk 4]
```diff
@@ -39,6 +39,12 @@
                     "fa_icon": "fas fa-chart-area",
                     "enum": [50, 75, 100, 150, 200]
                 },
+                "with_control": {
+                    "type": "boolean",
+                    "description": "Use controls.",
+                    "help_text": "Use this to indicate that your design file lists controls.",
```
Comment (JoseEspinosa) [inline nextflow_schema.json:45]:
```suggestion
                    "help_text": "Use this to indicate that your samplesheet lists controls.",
```

**Review by JoseEspinosa — APPROVED** [review]
LGTM! Just dropped some small suggestions :rocket
Don't know if @drpatelh would like to review this one before we merged it? 🤔

**Pull request merged**